overview.tt

```diff
@@ -0,0 +1,279 @@
+          old version and some files from the new version - which would be bad
+          because a program might crash if it's restarted during that period.
+        </p>
+        <p>
+          And since nothing is overwritten, the old versions are still there
+          after an upgrade. This means that you can rollback to the old
+          version. Rollbacks are very fast: it doesn't involve lots of files
+          having to be restored from copies.
+        </p>
+      </div>
+    </div>
+  </div>
+
+  <div>
+    <!-- hacking, fun, party -->
+    [% PROCESS svg path="site-styles/assets/gfx-overview-gears.svg" %]
+    <div>
+      <h2>
+        It is important to experiment.
+      </h2>
+      <ul>
+        <li>generations</li>
+        <li>nix store</li>
+      </ul>
+      <p>
+        Use tools that allow you to play with different software as
+        you did when the job wasn't waiting for you the next day.
+      </p>
+      <div>
+        <p>
+          Install new software with no risk of breaking anything down. Nix
+          prevents conflicts not only between multiple development environments,
+          but also with the operating system. Your work environment will stay
+          intact, but allow you to experiment in a safe way.
+        </p>
+        <p>
+          No need to turn your system upside down just because you want to test
+          some experimental and bleeding edge versions of software.
+        </p>
+      </div>
+    </div>
+  </div>
+
+  <div>
+    [% PROCESS svg path="site-styles/assets/gfx-overview-gears.svg" %]
+    <div>
+      <h2>
+        No surprises. Trust in your deployments.
+      </h2>
+      <p>
+        Surprises only belong to birthdays, not deployments. With Nix there
+        are only two possible outcomes. Your system was successfully upgraded
+        or upgrade did not happen.
+      </p>
+      <ul>
+        <li>nixos</li>
+        <li>atomic</li>
+        <li>generations</li>
+      </ul>
+      <div>
+        <p>
+          Nix will never leave your system in an unknown state. It puts you in
+          a complete control. The upgrade to new version of the system is
+          atomic, meaning that there is no point in time where your system
+          could be in a broken state. Some call it magic, Nix users call it the
+          thing that they can not live without.
+        </p>
+      </div>
+    </div>
+  </div>
+
+  <div>
+    [% PROCESS svg path="site-styles/assets/gfx-overview-gears.svg" %]
+    <div>
+      <h2>
+        Contribution is only a Pull request away.
+      </h2>
+      <p>
+        Community is at the core of Nix. Collaboration is encouraged by
+        providing low barrier to contribute and bots to guide contributors
+        through the process.
+      </p>
+      <ul>
+        <li>community</li>
+        <li>contribute</li>
+      </ul>
+      <div>
+        <p>
+          Development of Nix, Nix software collection and NixOS Linux
```

Comment:
```suggestion
          Development of Nix, the Nix packages collection and the NixOS Linux
```

BTW we haven't mentioned Nixpkgs explicitly, maybe we should since it's a big selling point.

Reply:
:+1: I will add the section about the nixpkgs